src/Discord/Provider.php

```diff
@@ -115,11 +115,13 @@ protected function formatAvatar(array $user)
     protected function mapUserToObject(array $user)
     {
         return (new User())->setRaw($user)->map([
-            'id'       => $user['id'],
-            'nickname' => $user['username'].($user['discriminator'] !== '0' ? '#'.$user['discriminator'] : ''),
-            'name'     => $user['username'],
```

Comment:
Removing the name field here would be a breaking change.

Reply:
Happy to dd the `username` field in addition,  but they can always use `getRaw` to get the other fields.